backend/tools/add_slot_from_db_for_schedule_admin.py

```diff
@@ -0,0 +1,26 @@
+import db.models as models
+from db.models import Slots, SlotsShema
+
+
+def add_slot_from_db_for_schedule_admin(start, end):
+    """Add new free slots in db for admin schedule.
+    If the slot is added - return json with this slots.
+    If the slot is don't added - return error msg
+    """
+    try:
+        slots = models.Slots(start_interval=start, end_interval=end)
+        models.db.session.add(slots)
+        models.db.session.commit()
+        return get_last_slot_id(slots.id)
+    except Exception:
+        models.db.session.rollback()
+        return {
```

Comment:
```suggestion
        raise
```

Reply:
i think better use return, becouse raise show 800 strings of information and not understand how use this information. Only if we need to write a new class for the exception, but i think return is eniught😀

Reply:
It looks all right for me. 

You should process the exception at some point. That means you maybe log the detailed information and give the user short summary. Then a developer can check the details from the logs.
The user shouldn't see all the details.

We would have to process the exception and convert it to a json message at some point. We either do it here or in `new_free_slots_for_admin`. 

One problem I see currently is that you are giving HTTP status OK when you have an error. That is not good. Fix it somehow.